package.json

```diff
@@ -11,6 +11,10 @@
     "node": ">=5"
   },
   "scripts": {
+    "prepublish": "npm run install-cli && npm run install-extension && npm run install-viewer",
```

Comment:
We've been led astray with `prepublish`:

<img width="708" alt="screen shot 2017-01-09 at 6 15 26 pm" src="https://cloud.githubusercontent.com/assets/238208/21791443/a22ac454-d697-11e6-9c72-9fcd952aa644.png">

In light of that deprecation, let's do these on `npm install` instead?

For  non-Travis node v4 users, we could create`setNodeHarmony.sh` that adds the `--harmony` flag if needed and run it in Travis and before the extension builds itself.